eclipse-ee4j/jaxrs-api
# TCK Migration : Move a set of jaxrs/ee/rs/ tests from jakartaee-tck

This is to migrate the tests from jakartaee-tck jaxrs/ee/rs/container,beanparam,constrainedto,delete folders

To test the changes :

export JAVA_HOME=<JDK11_HOME>
export M2_HOME=<MAVEN_HOME>
export PATH=$M2_HOME/bin:$JAVA_HOME/bin:$PATH

jaxrs-tck/
mvn clean install

jersey-tck/
mvn clean verify -Parq-glassfish-managed

The tests migrated :
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/beanparam
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/container
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/cookieparam
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/constrainedto
jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/delete


failing test :  ee.rs.container.requestcontext.security.getSecurityContextTest : will be tracked in https://github.com/eclipse-ee4j/jaxrs-api/issues/1020
